source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,188 @@
+
+
+Reviewing Secure Connection Status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following slash command to review the current status of all secure connections established for your Mattermost instance:
+
+``/secure-connection status``
+
+Status details include:
+- Connection ID
+- Connection URL
+- Description
+- Invite accepted (Yes/No)
+- Online (Yes/No)
+- Last ping timestamp (UTC)
+
+Accepting a Secure Connection Invitation
+----------------------------------------
+
+Use the following slash command to accept a secure connection invitation from a remote Mattermost instance:
+
+``/secure-connection accept --name --displayname --password --invite [code blob]``
+
+For example:
+
+``/secure-connection accept --name AcmeUS --displayname “AcmeUSA” --password examplepassword --invite [code-blob]``
+
+This slash command accepts a secure connection invitation from AcmeUS.
+
+Sharing Channels with Secure Connections
+----------------------------------------
+
+Within a specific Public or Private channel, use the following slash command to invite secure connections:
+
+``/share-channel invite --connectionID <--readonly>``
+
+You can extend an invitation that permits remote members to participate in the channel based on their channel and member permissions. 
+
+Alternatively, you can extend a read-only invitation to a secure connection by appending the optional --readonly parameter to this command. Remote members can’t post or reply to messages within shared read-only channels.
+
+.. tip:: 
+
+    You can re-extend an invitation to a secure connection by omitting the optional ``--readonly`` parameter that also enables remote member participation in that channel.
+
+For example:
+
+``/share-channel invite --connectionID "AcmeUS"``
+ 
+This slash command invites AcmeUS to the current channel.
+
+.. tip:: 
+    See `Reviewing Secure Connection Status <#reviewing-secure-connection-status>`_ to find the connectionID for a shared connection.
+
+Uninviting Shared Channel Connection from a Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within a specific channel, use the following slash command to uninvite a secure connection:
+
+``/share-channel uninvite --connectionID``
+
+This slash command removes a secure connection from the current channel based on its Connection ID. This secure connection may continue to be invited to other shared channels.
+
+Removing All Secure Connections from a Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within a specific channel, use the following slash command to remove all secure connections: 
+
+``/share-channel unshare``
+
+This slash command removes all secure connections from the current channel. Secure connections may continue to be invited to other shared channels. 
+
+While unsharing a shared channel stops synchronizing the channel with the other Mattermost server, the channel continues to function for local users as expected. 
```

Comment:
0/5, if we add it to both places, but this is applicable also to uninviting a specific connection from a channel as well from what I recall.